lib/Slic3r/GUI/Plater/SplineControl.pm

```diff
@@ -267,7 +267,11 @@ sub _update_canvas_size {
     my @size = ($canvas_w - 2*$padding, $canvas_h - 2*$padding);
     $self->{canvas_size} = [@size];
     
-    $self->{scaling_factor_x} = $size[0]/($self->{max_layer_height} - $self->{min_layer_height});
+    if ($self->{max_layer_height} > $self->{min_layer_height}) {
```

Comment:
I think you should still check for a minimum difference between max and min layer height so that we don't end up with silly numbers for scaling_factor_x when they're almost the same but still satisfy this condition.

Reply:
I think if the user is doing particularly silly things that are technically correct (here, it is technically correct from the operation of the software to have min layer 0.1 and max layer 0.100000000001),  they deserve to get technically correct yet silly outputs.

We check for max <= min via the config validation as well.